less/mixins.less

```diff
@@ -1 +1,32 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+@import 'global.less';
```

Comment:
You had already imported `global.less` in `index.less`. It means you have imported the file twice.